OracleGoldenGate/21c/README.md

```diff
@@ -55,9 +55,32 @@ Sending build context to Docker daemon
 Successfully tagged oracle/goldengate-for-bigdata:21.4.0.0.0   
 ```
 
+To create a container image for GoldenGate for MySQL Database, use the following script:
+
+```sh
+$ docker build --tag=oracle/goldengate-for-MySQL:21.3.0.0.0 \
+                --build-arg INSTALLER=213000_ggs_Linux_x64_MySQL_services_shiphome.zip .
+Sending build context to Docker daemon
+...
+Successfully tagged oracle/goldengate-for-MySQL:21.3.0.0.0
+```
+
+To create a container image for GoldenGate for PostgreSQL Database, use the following script:
+
+```sh
+$ docker build --tag=oracle/goldengate-for-PostgreSQL:21.3.0.0.0 \
+                --build-arg INSTALLER=213000_ggs_Linux_x64_PostgreSQL_services_shiphome.zip .
+Sending build context to Docker daemon
+...
+Successfully tagged oracle/goldengate-for-PostgreSQL:21.3.0.0.0
+
 ## Running Oracle GoldenGate in a Container
 
-Use the `docker run` command to create and start a container from the Oracle GoldenGate container image.
+The container image name for individual database is different. Use the below image name with `docker run` command to create and start a container from the Oracle GoldenGate container image.
```

Comment:
This needs re-work.

Just leave the 'Use the `docker run`...' line alone and use `<image-name>` on line 95, which you already have. Then add `<image-name>` to that list of parameters. with a brief description. Do *not* reference the database name, but instead use:

```md
* `<image-name>`    - The name of the previously created container image
```